src/view/PointsShortcode.php

```diff
@@ -251,12 +252,15 @@ private function get_optimistic_lock_value(array $keys)
             array_values($current_points));
 
 	    $current_optimistic_lock_value = array_reduce( $keys, function ( $carry, PointsKey $key ) use ( $points_by_key ) {
-		    $response = $points_by_key[ $key->question_id . self::FIELD_NAME_PART_SEP . $key->group_id ];
-		    if ( $response->created != null ) {
-			    return max( $carry, $response->created->getTimestamp() );
-		    }
+		    $temp_key = $key->question_id . self::FIELD_NAME_PART_SEP . $key->group_id;
 
-		    return $carry;
+		    $response_timestamp = isset( $points_by_key[ $temp_key ] )
```

Comment:
If the oneliner is not one line, use regular if.

Reply:
Fixed.